pkg/identityserver/user_registry.go

```diff
@@ -258,6 +259,37 @@ func (is *IdentityServer) getUser(ctx context.Context, req *ttnpb.GetUserRequest
 	return usr, nil
 }
 
+func (is *IdentityServer) listUsers(ctx context.Context, req *ttnpb.ListUsersRequest) (users *ttnpb.Users, err error) {
+	req.FieldMask.Paths = cleanFieldMaskPaths(ttnpb.UserFieldPathsNested, req.FieldMask.Paths, getPaths, nil)
+	authInfo, err := is.authInfo(ctx)
```

Comment:
We also have `is.IsAdmin` (in `entity_access.go`). Maybe nice to add a `is.RequireAdmin` (that returns an err) next to that.